From 619775c3cfd2bc8559abc4395bf7d85b72bd723f Mon Sep 17 00:00:00 2001
Message-Id: <619775c3cfd2bc8559abc4395bf7d85b72bd723f.1648737432.git.geliang.tang@suse.com>
From: Leah Neukirchen <leah@vuxu.org>
Date: Wed, 16 Dec 2020 11:03:06 +0100
Subject: [PATCH] bpf: Remove unnecessary <argp.h> include from
 preload/iterators
Git-commit: 619775c3cfd2bc8559abc4395bf7d85b72bd723f
Patch-mainline: v5.12-rc2
References: bsn#127
Modified-by-SEL: No


This program does not use argp (which is a glibcism). Instead include <errno.h>
directly, which was pulled in by <argp.h>.

Signed-off-by: Leah Neukirchen <leah@vuxu.org>
Signed-off-by: Daniel Borkmann <daniel@iogearbox.net>
Acked-by: Song Liu <songliubraving@fb.com>
Link: https://lore.kernel.org/bpf/20201216100306.30942-1-leah@vuxu.org
Signed-off-by: Alexei Starovoitov <ast@kernel.org>
Signed-off-by: Geliang Tang <geliang.tang@suse.com>
---
 kernel/bpf/preload/iterators/iterators.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/kernel/bpf/preload/iterators/iterators.c b/kernel/bpf/preload/iterators/iterators.c
index b7ff87939172..5d872a705470 100644
--- a/kernel/bpf/preload/iterators/iterators.c
+++ b/kernel/bpf/preload/iterators/iterators.c
@@ -1,6 +1,6 @@
 // SPDX-License-Identifier: GPL-2.0
 /* Copyright (c) 2020 Facebook */
-#include <argp.h>
+#include <errno.h>
 #include <stdio.h>
 #include <stdlib.h>
 #include <string.h>
-- 
2.34.1

